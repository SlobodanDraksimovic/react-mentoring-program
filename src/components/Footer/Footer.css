.footer-container {
  background-color: var(--secondary);
  padding: 1.5rem;
  margin-top: 0.5rem;
  width: 100%;
}

.footer-elements {
  list-style: none;
  margin: 0;
  padding: 0;
  text-align: center;
}

.footer-element {
  font-size: 1.25rem;
  line-height: 1.5rem;
  color: var(--primary);
  display: inline-block;
  margin: 0 1rem;
}

.footer-element .first-subpart {
  font-weight: 900;
}

.footer-element .second-subpart {
  font-weight: 500;
}
