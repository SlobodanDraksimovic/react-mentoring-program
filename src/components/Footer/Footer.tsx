import React from 'react';

import './Footer.css';

const Footer: React.FC = () => (
  <footer className="footer-container">
    <ul className="footer-elements">
      <li className="footer-element">
        <span className="first-subpart">netflix</span>
        <span className="second-subpart">roulette</span>
      </li>
    </ul>
  </footer>
);

export default Footer;
