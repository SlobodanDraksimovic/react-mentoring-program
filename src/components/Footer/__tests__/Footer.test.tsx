import React from 'react';
import { render, cleanup } from '@testing-library/react';

import Footer from '../Footer';

describe('<Footer />', () => {
  afterEach(cleanup);

  it('should render Footer', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
