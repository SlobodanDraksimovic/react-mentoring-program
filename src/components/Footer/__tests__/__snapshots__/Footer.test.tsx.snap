// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render Footer 1`] = `
<DocumentFragment>
  <footer
    class="footer-container"
  >
    <ul
      class="footer-elements"
    >
      <li
        class="footer-element"
      >
        <span
          class="first-subpart"
        >
          netflix
        </span>
        <span
          class="second-subpart"
        >
          roulette
        </span>
      </li>
    </ul>
  </footer>
</DocumentFragment>
`;
